.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to RIOT !
+
+### Preliminary checks
+
+The RIOT community takes a lot of care to contributions quality.
```

Comment:
It might be better to rephrase this as "The RIOT community cares a lot about code quality."